python-avocado.spec

```diff
@@ -148,6 +148,7 @@ popd
 %{__mkdir} -p %{buildroot}%{_mandir}/man1
 %{__install} -m 0644 man/avocado.1 %{buildroot}%{_mandir}/man1/avocado.1
 %{__install} -m 0644 man/avocado-rest-client.1 %{buildroot}%{_mandir}/man1/avocado-rest-client.1
+%{__install} -d -m 0755 %{buildroot}%{_sharedstatedir}/avocado/data
```

Comment:
Right.  Even after your comments, I don't see a problem with the current approach.  If you do, let's rework this.  If not, let it flow through.